main_test.go

```diff
@@ -740,46 +563,45 @@ OK
 }
 
 func TestLot(t *testing.T) {
-	t.Parallel()
+	// t.Parallel()
 	path := "_examples/lot"
 	testPkg(t, pkg{
 		path: path,
-		lang: []string{"py2", "py2-cffi", "py3-cffi", "pypy2-cffi", "pypy3-cffi"},
+		lang: features[path],
 		want: []byte(`l.SomeString : some string
 l.SomeInt : 1337
 l.SomeFloat : 1337.1337
 l.SomeBool : True
-l.SomeListOfStrings: []string{"some", "list", "of", "strings"}
-l.SomeListOfInts: []int64{6, 2, 9, 1}
-l.SomeListOfFloats: []float64{6.6, 2.2, 9.9, 1.1}
-l.SomeListOfBools: []bool{true, false, true, false}
+l.SomeListOfStrings: go.Slice_string len: 4 handle: 2 [some, list, of, strings]
+l.SomeListOfInts: go.Slice_int64 len: 4 handle: 3 [6, 2, 9, 1]
+l.SomeListOfFloats: go.Slice_float64 len: 4 handle: 4 [6.6, 2.2, 9.9, 1.1]
+l.SomeListOfBools: go.Slice_bool len: 4 handle: 5 [True, False, True, False]
 OK
 `),
 	})
 }
 
 func TestSlicePtr(t *testing.T) {
-	t.Parallel()
+	// t.Parallel()
 	path := "_examples/sliceptr"
 	testPkg(t, pkg{
 		path: path,
-		lang: []string{"py2", "py2-cffi", "py3-cffi", "pypy2-cffi", "pypy3-cffi"},
-		want: []byte(`sliceptr.IntVector{1, 2, 3}
-sliceptr.IntVector{1, 2, 3, 4}
-sliceptr.StrVector{"1", "2", "3", "4"}
+		lang: features[path],
+		want: []byte(`sliceptr.sliceptr_IntVector len: 3 handle: 1 [1, 2, 3]
+sliceptr.sliceptr_IntVector len: 4 handle: 1 [1, 2, 3, 4]
+sliceptr.sliceptr_StrVector len: 4 handle: 2 [1, 2, 3, 4]
```

Comment:
hum... do we really want this stuttering?

Reply:
I believe there are cases where having the package name in the type name is necessary..